.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What about these rules? Do you think we can apply any of these?
```js
{
  "printWidth": 80,
  "tabWidth": 2,
  "useTabs": false,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "all",
  "arrowParens": "avoid",
  "endOfLine": "auto"
}
```

Reply:
@lalitmee thanks for your rules, I have added them and formatted the code accordingly. The production build compiled without any issues